engine/src/main/java/org/terasology/rendering/dag/DefaultStateChangeStorage.java

```diff
@@ -15,20 +15,19 @@
  */
 package org.terasology.rendering.dag;
 
-import java.util.List;
+import java.util.HashMap;
+import static org.lwjgl.opengl.GL11.GL_FILL;
 
 /**
- *
+ * TODO: Add javadocs
  */
-public class RenderPipelineExecutor {
-
-    private List<PipelineTask> taskList;
+class DefaultStateChangeStorage extends HashMap<Class<? extends StateChange>, StateChange> {
```

Comment:
No problem, I thought about possibility of cases where having multiple StateChange instances with same copies of default values would be bad. Shame on Java that we can propose a static method for this. 

Let me make it as you mentioned.